docs/release/trg-2/trg-2-3.md

```diff
@@ -21,17 +20,22 @@ All repositories **must** contain the following files and folders:
 ```shell
 /docs
 /charts
-AUTHORS.md
 CODE_OF_CONDUCT.md
 CONTRIBUTING.md
 DEPENDENCIES
 LICENSE
 NOTICE.md
 README.md
-INSTALL.md
 SECURITY.md
 ```
 
+Recommended but optional:
+
+```shell
+AUTHORS.md
+INSTALL.md
```

Comment:
hey @FaGru3n, I just split the listing into two in order to distinguish mandatory and optional, not sure if it makes sense to comment more and overload it since down below are refrences describe specifically README and INSTALL in details

Reply:
IMHO comment in the code section would be one solution but i am also fine with your idea to describe it directly under the code section or maybe you can also reference "README" TRG 1.01

Reply:
sorry missed that part just whatched only the changings...